### PR TITLE
VG Vending 2: Free Market Boogaloo

### PR DESCRIPTION
**Pull request by TaxMaster**
Adds VG clothing back into the server proper with a free market twist.

The main difference between the last PR and this one is that this one tries to strike a fine balance between laissez-faire and banning everything. It (tries) to accomplish this by having the offending clothing not spawn in vending machines as originally intended, but having to be ordered from cargo.
They are fairly expensive as well, with the cheapest crate costing 1000 credits more than what the biker crate costs, with the reenactor crate (read: Nazi clothing + soviet clothing) being just shy of the most expensive crate in cargo’s roster.
It also removes the turtlenecks from vg_clothes and adds them to its own separate clothing file along with updating the dm file in cit_uniforms, because that was just a bad idea.

🆑 maint bar
add: maint bar
/🆑

### CODE REVIEW

File: code/modules/clothing/head/vg_hats.dm [hunk 1]
```diff
@@ -29,6 +30,7 @@
 	icon_state = "biker_helmet"
 	icon = 'icons/obj/clothing/vg_clothes.dmi'
 	flags_inv = HIDEMASK|HIDEEARS|HIDEEYES|HIDEFACE|HIDEHAIR
+	icon = 'icons/obj/clothing/vg_clothes.dmi'
```
Comment (Poojawa) [inline code/modules/clothing/head/vg_hats.dm]: You've basically put the same code twice.

File: code/modules/clothing/head/vg_hats.dm [hunk 2]
```diff
@@ -37,6 +39,7 @@
 	icon_state = "richard"
 	icon = 'icons/obj/clothing/vg_clothes.dmi'
 	flags_inv = HIDEMASK|HIDEEARS|HIDEEYES|HIDEFACE|HIDEHAIR
+	icon = 'icons/obj/clothing/vg_clothes.dmi'
```
Comment (Poojawa) [inline code/modules/clothing/head/vg_hats.dm]: and here

File: code/modules/clothing/head/vg_hats.dm [hunk 3]
```diff
@@ -45,6 +48,7 @@
 	item_state = "megahelmet"
 	icon = 'icons/obj/clothing/vg_clothes.dmi'
 	siemens_coefficient = 1
+	icon = 'icons/obj/clothing/vg_clothes.dmi'
```
Comment (Poojawa) [inline code/modules/clothing/head/vg_hats.dm]: and here

File: code/modules/clothing/head/vg_hats.dm [hunk 4]
```diff
@@ -80,6 +87,7 @@
 	icon = 'icons/obj/clothing/vg_clothes.dmi'
 	armor = list(melee = 50, bullet = 40, laser = 40,energy = 40, bomb = 5, bio = 0, rad = 0)
 	siemens_coefficient = 1
+	icon = 'icons/obj/clothing/vg_clothes.dmi'
```
Comment (Poojawa) [inline code/modules/clothing/head/vg_hats.dm]: etc

File: code/citadel/cit_uniforms.dm [hunk 5]
```diff
@@ -6,6 +6,7 @@
 	item_color = "bb_turtle"
 	body_parts_covered = CHEST|ARMS
 	can_adjust = 1
+	icon = 'icons/obj/clothing/turtlenecks.dmi' //Seriously, who thought it was a good idea to throw it into vg_clothes? -TaxMaster
```
Comment (Poojawa) [inline code/citadel/cit_uniforms.dm]: This comment isn't needed.

File: code/modules/clothing/head/vg_hats.dm [hunk 6]
```diff
@@ -53,6 +57,7 @@
 	item_state = "protohelmet"
 	icon = 'icons/obj/clothing/vg_clothes.dmi'
 	siemens_coefficient = 1
+	icon = 'icons/obj/clothing/vg_clothes.dmi'
```
Comment (Poojawa) [inline code/modules/clothing/head/vg_hats.dm]: and here

File: code/modules/clothing/head/vg_hats.dm [hunk 7]
```diff
@@ -61,6 +66,7 @@
 	item_state = "megaxhelmet"
 	icon = 'icons/obj/clothing/vg_clothes.dmi'
 	siemens_coefficient = 1
+	icon = 'icons/obj/clothing/vg_clothes.dmi'
```
Comment (Poojawa) [inline code/modules/clothing/head/vg_hats.dm]: and here

**Comment by TaxMaster** [comment]
Unless you find more code-related problems, it seems all right to me.

File: code/modules/clothing/head/vg_hats.dm [hunk 8]
```diff
@@ -13,6 +13,7 @@
 	flags = STOPSPRESSUREDMAGE
 	strip_delay = 80
 	dog_fashion = null
+	icon = 'icons/obj/clothing/vg_clothes.dmi'
```
Comment (Poojawa) [inline code/modules/clothing/head/vg_hats.dm]: the one too,

**Comment by TaxMaster** [comment]
Alright, I looked over the code for all of the /vg/ clothing dms and they seem fine so unless i'm blind, it good(?) now.

**Pull request merged**